tools/myst/host/package.c

```diff
@@ -767,28 +791,31 @@ int _exec_package(
 
     if (_is_null_rootfs(details->rootfs.buffer, details->rootfs.buffer_size))
     {
-        char* env;
-
-        if (!(env = getenv("MYST_ROOTFS_PATH")))
+        /* if rootfs has not already been specified by --myst-rootfs=<path> */
+        if (*options.rootfs == '\0')
         {
-            fprintf(stderr, "MYST_ROOTFS_PATH is undefined\n");
-            goto done;
-        }
+            char* env;
```

Comment:
Suggest renaming to "roothash_path"

Reply:
This environment variable really does refer to the rootfs path (not a roothash). This is what allows packaging mode to find an ext2 rootfs.